File: 민균/2주차/code_1978.java
package baekjoon.study.week2;

import java.util.Scanner;

public class code_1978 {

    public static boolean isPrime(int num){
        if (num == 1) return false;
        for (int i = 2; i <num; i++){
            if (num%i==0) return false;
        }
        return true;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int []arr = new int[N];
        int cnt = 0;
        for (int i = 0; i<N; i++){
            arr[i] = sc.nextInt();
            if(isPrime(arr[i])) cnt++;
        }
        System.out.println(cnt);
        sc.close();
    }
}

File: 혜원/week2/code_1978.cpp
#include <iostream>
using namespace std;

int main() {
	int N = 0, cnt = 0, res = 0;
	int arr[1001] = { 0 };

	cin >> N;
	if (N > 100) return 0;

	for (int i = 0; i < N; i++) {
		cin >> arr[i];
		for (int j = 1; j <= arr[i]; j++) {
			if (arr[i] % j == 0) cnt++;
		}
		if (cnt == 2) res++;
		cnt = 0;
	}
	cout << res;

	return 0;
}

////소수를 구하는 방법 다시 보자.